src/middlewared/middlewared/alert/source/volume_rekey.py

```diff
@@ -5,8 +5,9 @@
 
 class VolumeRekeyAlertSource(FilePresenceAlertSource):
     level = AlertLevel.CRITICAL
-    title = ("Encrypted volume failed to rekey some disks. Please make "
-             "sure you have working recovery keys, check logs files and "
-             "correct the error as it may result to data loss.")
+    title = "Failed to Rekey One or More Disks of Encrypted Pool"
+    text = ("Rekeying one or more disks in an encrypted pool failed. Please make "
+            "sure working recovery keys are available, check log files, and "
```

Comment:
"check log files" --> could this be enhanced further to aid the user? Specific file names, path to where the log files are stored, that sort of thing?

Reply:
@william-gr do you know which log files?

Reply:
@themylogin geli_rekey function will use `log.error` so the log message of why the rekey failed will be in `/var/log/messages`